bin/playwright.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# todo: find a way to make this portable for Windows users so it can be used for the package.json script
```

Comment:
I wonder if we can re-write this script using Node/JS  to make it cross-platform? Not for this PR :)

Reply:
We could... node's sub-process control is pretty weird, I could never really figure out how best to manage them.